openfold/np/protein.py

```diff
@@ -156,7 +160,7 @@ def from_pdb_string(pdb_str: str, chain_id: Optional[str] = None) -> Protein:
                 chain_id += 1
 
     unique_chain_ids = np.unique(chain_ids)
-    chain_id_mapping = {cid: n for n, cid in enumerate(string.ascii_uppercase)}
+    chain_id_mapping = {cid: n for n, cid in enumerate(get_chain_tags())}
```

Comment:
I don't remember this change, perhaps it's based on getting changed from the public repo. Is this something that you added/modified ?

Reply:
I made this change, as in some sabdab entries chain_ids were lowercase and even numbers.